creator-node/src/routes/files.js

```diff
@@ -111,7 +107,7 @@ const getCID = async (req, res) => {
     // Cat 1 byte of CID in ipfs to determine if file exists
     // If the request takes under 500ms, stream the file from ipfs
     // else if the request takes over 500ms, throw an error
-    await ipfsSingleByteCat(CID, req, 500)
+    await ipfsSingleByteCat(CID, req.logContext, 500)
```

Comment:
do we still need to await a single byte cat anywhere? i would expect this to actually not be necessary since the rehydrate operations are now enqueued

Reply:
https://github.com/AudiusProject/audius-protocol/blob/7c159b811200cf6a77fdcf474c995733738315df/creator-node/src/utils.js#L178

Reply:
so in this flow, if streaming from fs fails, it streams from IPFS. we can't ensure that the rehydrate occurred before this ipfsSingleByteCat call, and I think this line is a good check to ensure that it is present in ipfs

Reply:
right but a single byte cat failing used to trigger a rehydrate operation - now if it fails there is no specific action to be taking and we serve anyway potentially leading to a timeout in both cat and the stream operation. maybe we add a timeout to the final stream operation instead? or alternatively we should be issuing a rehydrate if and only if a single byte cat fails (this is handled already in the rehydrate function)

Reply:
@dmanjunath thoughts here?

Reply:
after some discussion with @SidSethi, this is a no-op let's get this in

Reply:
would this go to the catch block if it times out and send response? @hareeshnagaraj @vicky-g

Reply:
if ipfsSingleByteCat() fails, it would go to the catch block and send an error response.

Reply:
if im understanding correctly, you're saying that if we add a timeout to the cat and/or stream logic, we can ensure that the prior rehydrate call succeeded and it is now in ipfs. 

For this particular flow, this is how it currently works:
I1. async rehydrate + stream from fs
2. if that fails, stream from ipfs (we are assuming it is already in ipfs)
3. if that fails, send error response

I think that we shouldn't be dependent on the rehydrate and just fail quickly if the file fails to be streamed from fs and cannot be found in ipfs? @dmanjunath @hareeshnagaraj